### PR TITLE
Add link to create project via the Equip sample project to docs

### PR DESCRIPTION
**Pull request by sctape**
I know we already have a link to the sample project in the github README for `equip/framework` but I thought it would be helpful to also include it in the `equipframework.readthedocs.io` docs as well.

### CODE REVIEW

**Comment by shadowhand** [comment]
:+1: I'm sure this has been missed by new users in the past.

File: docs/index.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ Subsequent examples will assume your project has a directory structure similar t
     └── index.php
 ```
 
+Instead of installing Equip via Composer, you can get going a little faster by using the [sample project](https://github.com/equip/project) that shows the best way to use this framework.
```
Comment (bvisness) [inline docs/index.md]: Do we want to say "the best way" or just "a good way"?

Comment (sctape) [inline docs/index.md]: I like that language change. I had originally stolen this line from the project README.

**Comment by sctape** [comment]
@bvisness @shadowhand had to sync with master. Can I get a thumb when you have a moment?

**Comment by shadowhand** [comment]
👍

**Pull request merged**